lib/session.js

```diff
@@ -28,11 +28,9 @@ class Speed {
   compute () {
     const time = now()
     const currentMinute = time - time % 60
-    return Range(0, this.size)
-      .map(n => {
-        return this.counters[currentMinute - n * 60]
-      })
-      .filter(c => c !== undefined)
+    let speed = Range(0, this.size).map(n => this.counters[currentMinute - n * 60])
+    speed = dropWhile(speed.reverse(), v => v === undefined)
```

Comment:
I think this can return wrong results in some cases.
For example : 
At `currentMinute - 16` we have a speed of 5
On the range `currentMinute - 15` -> `currentMinute - 5` we have a speed of 0 (so undefined)
On the range `currentMinute - 4 ` -> `currentMinute` we have a speed of 5
This will return only the latest 4 minutes, whereas technically we have data for 16 minutes, and we are dropping information of the `0 speed`.
But maybe it's not so important?

Reply:
You're right. I'm going to look at another way to do it.